dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,24 +1,16 @@
 module SurveyResultsHelper
   def show_survey?
-    # rubocop:disable Lint/UnreachableCode
-
-    # Disabled for all users on 2016-03-30 after being shown for thirty days.
-    return false
```

Comment:
Can we put a counter that automatically disables it after 30 days or force it to disable at a certain date?